op-scim-bridge/templates/horizontalPodAutoscaler.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.scim.autoscaling.enabled }}
```

Comment:
should these sub-template/partial template filename be prefixed with an underscore. I see we do that with the helper but not these.

Reply:
actually ignore that.... It seems like these are standalone resources in their own right.